src/viur/core/prototypes/singleton.py

```diff
@@ -192,7 +192,7 @@ def canPreview(self) -> bool:
         if user["access"] and "root" in user["access"]:
             return True
 
-        if user["access"] and "%s-edit" % self.viewSkel.kindName in user["access"]:
+        if user["access"] and f"{self.viewSkel.kindName}-edit" in user["access"]:
```

Comment:
This is inconsistent with all other places, and also a bug that never occured yet because Singletons are usually only edited by root-users...

```suggestion
        if user["access"] and f"{self.moduleName}-edit" in user["access"]:
```